### ISSUE TITLE
`aura` returns `Permission denied (publickey)` when fetching packages from AUR

### ISSUE THREAD

**Issue by christarazi** [fosskers/aura#822]
```
aura >>= Building python-pipenv-git...
Cloning into bare repository '/var/cache/aura/vcs/python-pipenv-git/python-pipenv-git/python-pipenv-git'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
==> ERROR: Failure while downloading python-pipenv-git git repo
    Aborting...

aura >>= There was a makepkg failure.
```

I am running aura like 

```
sudo -E aura --hotedit --aursync --delmakedeps --noconfirm --sysupgrade
```

If I run `ssh -vT git@github.com` in my regular shell, I can see that GitHub accepts the authentication. If I do `env -i -- ssh -vT git@github.com`, then I get the same message as the above. 

It seems that `aura` is clobbering other environment variables which are relevant to my SSH authentication. Any advice here?

**Comment by fosskers** [comment]
I just released `4.0.0-alpha10` which provides `--noconfirm` for `-A`.

**Comment by fosskers** [comment]
Hi there, as this should not longer be occurring, I'm going to close this issue. Aura 4 will be released soon.
